book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] Look at these lines of code from the initialize block of your Rental class.

    ```ruby
      @book = book
      book.rentals << self
    ```

  These two lines tell us the Rental class takes a book and adds itself to the book's rental collections to achieve the 'belongs-to' association.
  
  Now, over here in the Book class, we only need to create a new rental by utilizing the Rental class to stay true to this association. We do not need to push any other rentals in this class, the Rental class takes care of that already. See below 👇🏼:
  
  ```ruby
    def add_rental(person, date)
      Rental.new(date, self, person)
    end
    ```
    
    Please work on this.